storagenode/pieces/store_test.go

```diff
@@ -446,7 +442,7 @@ func TestTrashAndRestore(t *testing.T) {
 				verifyPieceData(ctx, t, tStore, satellites[0].satelliteID, piece.pieceID, filestore.MaxFormatVersionSupported, lastFile.data, piece.expiration, publicKey)
 			} else {
 				// Expect the piece to be missing, it should be removed from the trash on EmptyTrash
-				r, err := store.Reader(ctx, satellites[1].satelliteID, piece.pieceID)
+				r, err := store.Reader(ctx, satellites[0].satelliteID, piece.pieceID)
```

Comment:
This was a fun one. A bug accidentally uncovered by my patch. Seems that `piece.trashDur` is unused and can be cleanly removed.